client/src/components/Profile/useStyles.ts

```diff
@@ -0,0 +1,98 @@
+import { Theme } from '@mui/material/styles';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    minHeight: '100vh',
+    '& .MuiInput-underline:before': {
+      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
+    },
+    justifyContent: 'center',
+  },
+  form: {
+    width: '100%',
+    marginTop: theme.spacing(9),
+  },
+  label: {
+    fontSize: 15,
+    color: '#000000',
+    paddingLeft: '5px',
+    marginTop: '30px',
+    textAlign: 'right',
+    marginBottom: '32px',
+  },
+  dropDown: {
+    marginLeft: '20px',
+    width: '20%',
+    height: '65px',
+  },
+  dropDownDate: {
+    marginLeft: '20px',
+    width: '15%',
+    height: '65px',
+  },
+  dropDownYear: {
+    marginLeft: '20px',
+    width: '15%',
+    height: '65px',
+  },
+  phoneNumber: {
+    fontSize: '15px',
+    marginLeft: '22px',
+    fontStyle: 'italic',
+    float: 'left',
+    textTransform: 'none',
+  },
+  gridItem: {
+    marginTop: '10px',
+    marginBottom: '10px',
+    textAlign: 'left',
+  },
+  phoneButton: {
+    fontSize: '15px',
+    marginLeft: '210px',
+    border: '0.5px solid #f14140',
+    borderRadius: '5px',
+    textTransform: 'none',
+    width: '30%',
+    backgroundColor: 'white',
+    color: '#f14140',
```

Comment:
You can apply these styles  as props to the`Button` component

Eg: `<Button color="primary" variant="outlined"....>`